Formula/rubyfmt.rb

```diff
@@ -1,25 +1,39 @@
 class Rubyfmt < Formula
   desc "Ruby autoformatter"
   homepage "https://github.com/penelopezone/rubyfmt"
-  url "https://github.com/penelopezone/rubyfmt/archive/v0.2.0.tar.gz"
-  sha256 "68ebc0fd30933b1e27b609cc34c69a3bc886747d11c5b949e460ce01814adaeb"
   license "MIT"
 
+  stable do
+    url "https://github.com/penelopezone/rubyfmt.git",
+    tag:      "v0.8.0",
+    revision: "ed99cc4586a908c97f8b19ed78801342f7aa8512"
+  end
+
   bottle do
-    sha256 cellar: :any_skip_relocation, arm64_big_sur: "93a5eab5b30e0ff0728557a9f51ce501cc9454fb7c6616b89f5a41d7be96e493"
-    sha256 cellar: :any_skip_relocation, big_sur:       "bcdd675a98a38a40e6fa8a7bbb780524665c863bd3fefda5e222d952363630a2"
-    sha256 cellar: :any_skip_relocation, catalina:      "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
-    sha256 cellar: :any_skip_relocation, mojave:        "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
-    sha256 cellar: :any_skip_relocation, high_sierra:   "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "ac11feb657dbe095a59e0bbbe1e8adbd8fe5eaa6dd989e36204780697b200e92"
-    sha256 cellar: :any_skip_relocation, all:           "ac11feb657dbe095a59e0bbbe1e8adbd8fe5eaa6dd989e36204780697b200e92"
+    sha256 cellar: :any_skip_relocation, arm64_ventura:   "6fc9e1228716d8eda62a4919dc70ba86a71d57930b9ca9114aa8bff56da85e12"
+    sha256 cellar: :any_skip_relocation, arm64_monterey:  "770767175d90b1b0ce0efba7e0ed89a9455a1c3b7597a2121215175fc7d40918"
+    sha256 cellar: :any_skip_relocation, ventura:         "6fc9e1228716d8eda62a4919dc70ba86a71d57930b9ca9114aa8bff56da85e12"
+    sha256 cellar: :any_skip_relocation, monterey:        "fb5ee30ee2c08e317bf17a36e2219bd06a9be8ce41551a769e64c6d79ea8d7cb"
+    sha256 cellar: :any_skip_relocation, big_sur:         "bcdd675a98a38a40e6fa8a7bbb780524665c863bd3fefda5e222d952363630a2"
+    sha256 cellar: :any_skip_relocation, catalina:        "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
+    sha256 cellar: :any_skip_relocation, mojave:          "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
+    sha256 cellar: :any_skip_relocation, high_sierra:     "8d9ed80d496220e02b9df146c41870079116cf798ab90734212d3cdc6080bb8b"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:    "ac11feb657dbe095a59e0bbbe1e8adbd8fe5eaa6dd989e36204780697b200e92"
```

Comment:
Leave the bottle block untouched, it is built automatically in CI.

Reply:
I'm assuming the CI step will handle adding/removing OSes as well then? A fix for Big Sur arm probably won't be in this particular release, so I assume my branch build will fail because of that.